spec/features/admin/admin_users_spec.rb

```diff
@@ -54,9 +56,9 @@
       click_button "Create user"
       user = User.find_by(username: 'bang')
       email = ActionMailer::Base.deliveries.last
-      email.subject.should have_content("Account was created")
-      email.text_part.body.should have_content(user.email)
-      email.text_part.body.should have_content('password')
+      expect(email.subject).to have_content("Account was created")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.